### DISCUSSION TITLE
空气净化器实体适配

### DISCUSSION THREAD

**Discussion by HoTK1** [XiaoMi/ha_xiaomi_home#129] [Ideas]
当前只能被识别为switch开关，希望整合成air Purifier

**Comment by SusanPhevos** [comment]
Home Assistant 没有 Air Purifier  https://github.com/home-assistant/architecture/discussions/1009

**Reply by HoTK1** [reply]
我描述有问题，应该说是桥接到homebridge上识别为Air Purifier，之前使用homebridge-miot可以实现

**Comment by Min9X1n** [comment]
可不可以整合部分至Fan呢？整合至Fan后能够使用的卡片也会多一些，Fan卡片+传感器卡片，是不是会好些

**Reply by HoTK1** [reply]
不能到fan，miotauto就是整合到fan了，用siri开关风扇会触发

**Comment by CODPHISH** [comment]
+1 ，现在是4个开关+1个风扇挡位的样子，homekit原生是支持air purifier类型配件的
